### PR TITLE
Goodbye CI

### PR DESCRIPTION
**Pull request by kevindweb**
Remove all CI files from the openNetVM repository.

## Summary:
You can see the new CI code is now at https://github.com/sdnfv/onvm-ci. It's private so I'm not sure who can see it, but @dennisafa and @koolzz should definitely get access to this. Ben will be helping me with CI moving forward with some cool new functionality. I will still keep the same OpenNetVM best practices in that repo. It just is easier to update CI when it doesn't have to get reviewed as much as the C code here. 

**Usage:**
See that all `/ci` files are gone.

| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          |  
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   |  
| Bug fixes                |
| New functionality        |
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  
| Dependency updates       |  👍 
| Web stats updates        | 

## Merging notes:
 - Dependencies: None  

**TODO before merging :**
 - [ ] PR is ready for review

## Test Plan:
Basically CI will test itself, but we should run things just to make sure there's nothing weird going on.

## Review: 
(optional) << @-mention people who should review these changes >>
@dennisafa 

(optional) **Subscribers:** << @-mention people who probably care about these changes >>

### CODE REVIEW

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Your results will arrive shortly

**Comment by kevindweb** [comment]
This is a farewell to CI. I have moved CI manager to a virtual machine, and all the code is working on the new repository. There is no need to make any more pull requests after this related to CI in this repo. CI has been updated to use Ubuntu 18 and Tokens,  which is why I removed the other pull request earlier.

My shellcheck_updates pull request does not include any `ci` files so there should be no conflict.

@bdevierno1 You should take a look at this. RIP CI, you've served the repo well.

**Comment by kevindweb** [comment]
@onvm final test sorry for the emails

**Comment by onvm** [comment]
> @onvm final test sorry for the emails

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> Time elapsed test

## CI Message

Your results will arrive shortly
